src/main/java/com/epam/pages/popup/AcademicClassPopup.java

```diff
@@ -1,6 +1,48 @@
 package com.epam.pages.popup;
 
-import com.epam.pages.common.CommonPopup;
+import com.epam.helpers.ErrorMessagesProvider;
+import com.epam.helpers.SharedTestData;
+import com.epam.helpers.UserDataProvider;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 
-public class AcademicClassPopup extends CommonPopup {
+import java.util.List;
+
+public class AcademicClassPopup extends CreatePopup {
+    @FindBy(id = "classNumber")
+    private WebElement classNumber;
+    @FindBy(xpath = "//input/following-sibling::div[@class='error']")
+    protected WebElement errorMessagesOfExistedClass;
```

Comment:
Delete this field, because in the CreatePopup class it is already defined.

Reply:
As long as the class extends from CommonPopup (I changed it and resolved comment below), i can't remove that field, because in CommponPopup we don't have such field defined

Reply:
ok